programs/conditional_vault/src/state/question.rs

```diff
@@ -0,0 +1,39 @@
+use super::*;
+
+/// Questions represent statements about future events.
+///
+/// These statements include:
+/// - "Will this proposal pass?"
+/// - "Who, if anyone, will be hired?"
+/// - "How effective will the grant committee deem this grant?"
+///
+/// Questions have 2 or more possible outcomes. For a question like "will this
+/// proposal pass," the outcomes are "yes" and "no." For a question like "who
+/// will be hired," the outcomes could be "Alice," "Bob," and "neither."
```

Comment:
1) Alice and Bob could be hired, so this is a false choice fallacy. Proposers need to be cautious with mutually exclusive outcomes. If they are in the same proposal we must have markets for the Cartesian product of possible states.

2) Maybe every multi-option decision market should have an "other" option as a catch-all. Or state clearly state in the proposal that if another outcome happens all markets will be treated as non-canonical.

Reply:
1. I'm saying it *could* be the outcomes (in the case where you only have enough money to hire one, say), not that it always would
2. what would be an example of where you'd want an 'other'?

Reply:
I was basing it just off etiquette in Reddit polls. But here's a thought experiment:

A DAO has a decision market to on whether a grant will be considered effective or not, by a committee, in six months. If the committee does not respond within seven months maybe the 'other' option should be declared the fallback outcome by the oracle and (some of?) the proposers’ AMM liquidity should be kept by the DAO as a penalty.

On further thought asking the question correctly solves this. E.g. Will the committee decide this grant is effective in 6 months. All other outcomes are encapsulated by the no. See [Drift's "Will Donald Trump win the 2024 Presidential Election?" [Yes] or [No]](https://app.drift.trade/bet/TRUMP-WIN-2024-BET)

However if there are >2 distinct outcomes that a proposers wants to be traded, I think "other" has to come into play. See polymarket election winner:

<img width="865" alt="image" src="https://github.com/user-attachments/assets/77ca77b7-87a9-47fe-8916-6af4716844cc">

Not sure if nerd sniping or helping here, feel free to resolve the thread.